.github/workflows/build.yml

```diff
@@ -10,8 +10,6 @@ name: build
 on:
   push:
 
-
-
 jobs:
   build:
```

Comment:
This should more explicit to call the job `deploy` for example

Reply:
Should we sort out the naming a bit later? I agree, but I would be tempted to have a separate `deploy` workflow that is triggered from this one completing, as we discussed last week, and I would be keen to get this change delivered before we head down that path.

Reply:
Done - Job renamed to "deploy_to_paas"